app/src/main/webapp/scripts/groups/config.js

```diff
@@ -78,7 +80,8 @@ angular.module('universeadm.groups.config', ['ui.router',
           group: function(groupService, $stateParams){
             var name = $stateParams.name;
             return name !== null && name.length > 0 ? groupService.get(name) : null;
-          }
+          },
+          $title: function() { return 'Edit Group | User Management'; }
```

Comment:
This is even used by the create group view. Create a new state for the `/group/` path with `Create Group | User Management` title.